app.py

```diff
@@ -66,7 +66,7 @@ def get_quote_by_category(category_id):
         category= list(mongo.db.categories.find()),
         categories=mongo.db.categories.find_one({'_id': ObjectId(category_id)}),
         quotes=list(mongo.db.quotes.find()),
-        sources=list(mongo.db.sources.find()) # tried to delete this after having extended the base.html but does not work
+        sources=list(mongo.db.sources.find()) 
         )
 
```

Comment:
tried to delete the "useless lists" from the different routes but without them the app does not work properly. Even after having extended the base.html instead of the home.html the issue persisted